modules/form-field/react/stories/stories_TextInput.tsx

```diff
@@ -206,4 +206,59 @@ storiesOf('Components|Inputs/Text Input/React/Visual Testing', module)
         )}
       </ComponentStatesTable>
     </StaticStates>
+  ))
+  .addParameters({
```

Comment:
Something I noticed is that this gets overridden in `Docs` and not quite sure how to solve for that yet..

Reply:
🤔 How about we just [remove the docs tab](https://github.com/storybookjs/storybook/blob/1e53a3ae7ab3d1b95fcc191087811f6a8e8a455c/examples/official-storybook/stories/addon-docs/addon-docs.stories.js#L51) for these stories? We could probably do it for all visual testing stories.

Reply:
So i tried this approach by this might be in 5.3?

Reply:
Okay let's just wait until #416 goes in 👍